packages/manager/modules/dedicated-universe-components/src/config/constants.config.js

```diff
@@ -0,0 +1,1333 @@
+import defaults from 'lodash/defaults';
+
+const constants = {
+  EU: {
```

Comment:
I don't know if constants should be extracted from their universes in a module with components specific to a universe

Reply:
These constants are used in multiple modules. What do you suggest we do ?